lib/file.sh

```diff
@@ -98,21 +98,26 @@ function _Dbg_resolve_expand_filename {
       # Try using cwd rather that Dbg_init_cwd
       full_find_file=$(_Dbg_expand_filename "$find_file")
     fi
+    if [[ ! -z "$full_find_file" ]]; then
+	_Dbg_file2canonic["$find_file"]="$full_find_file"
+    fi
     echo "$full_find_file"
     return 0
   else
     # Resolve file using _Dbg_dir
     typeset -i n=${#_Dbg_dir[@]}
     typeset -i i
     for (( i=0 ; i < n; i++ )) ; do
-      typeset basename="${_Dbg_dir[i]}"
+      typeset dirname="${_Dbg_dir[i]}"
       if [[  "$basename" == '\$cdir' ]] ; then
-	basename=$_Dbg_cdir
+	dirname=$_Dbg_cdir
       elif [[ "$basename" == '\$cwd' ]] ; then
-	basename=$(pwd)
+	dirname=$(pwd)
```

Comment:
quotes?

Reply:
Yes, you are correct should be handled now.